modules/distribution/pom.xml

```diff
@@ -585,6 +585,7 @@
                 <artifactId>maven-site-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
+
```

Comment:
is this intended?

Reply:
Fixed by https://github.com/wso2/product-is/pull/20366/commits/c810f9eac0b461388e620fa98005f7ab9e66d9c6
Should be a mistake